server/datastore/psql/create_maintenance_requests.go

```diff
@@ -0,0 +1,31 @@
+package psql
+
+import (
+    //"time"
+    "github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+func (db *DB) CreateMaintenanceRequests(request datastore.MaintenanceRequest) (datastore.MaintenanceRequest, error) {
+    insert, err := db.connection.Query("INSERT INTO maintenance_requests (request, user_submitted, date_submitted) VALUES ($1, $2, $3)", request.Request, request.UserSubmitted, request.DateSubmitted)
+    if err != nil {
+        return request, err
+    }
+    insert.Close() // Inserts the new request into the database
+
+    row, err := db.connection.Query("SELECT * FROM maintenance_requests") // Return the struct of the newly created request
```

Comment:
Rather than doing another query, you can use `RETURNING id` in the query.

https://stackoverflow.com/questions/33382981/go-how-to-get-last-insert-id-on-postgresql-with-namedexec

Reply:
fix: removed second query and returned the last inserted ID